tine20/Addressbook/Convert/Contact/VCard/Evolution.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Tine 2.0
+ *
+ * @package     Addressbook
+ * @subpackage  Convert
+ * @license     http://www.gnu.org/licenses/agpl.html AGPL Version 3
+ * @author      Lars Kneschke <l.kneschke@metaways.de>
```

Comment:
feel free to add you as an author here, I guess you aren't lars kneschke, for new files you are free to replace it.